data/src/test/java/com/example/data/dashboard/cache/CurrentDateTest.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.data.dashboard.cache
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class CurrentDateTest {
+
+    @Test
+    fun test() {
+        var currentDate = CurrentDate.Base(format = "yyyy")
```

Comment:
val actual = CurrentDate.Base(LocalDateTime.of(2024, 2, 28, 21, 15, 0), format="dd/mm/yyyy HH:mm:ss").date()
val expected = "28/2/2024 21:15:00"